truss/templates/control/requirements.txt

```diff
@@ -1,5 +1,5 @@
 dataclasses-json==0.5.7
-truss==0.4.8
+truss==0.4.10rc2
```

Comment:
[ignore] hmmm. odd that it was released before this PR is merged, but we'll roll with this. I can't wait to monorepo

Reply:
I pushed an rc version (`0.4.10rc2`) from this branch directly through to pypi to verify that the control server works with the new truss module changes – let me run the integration tests on this branch before merging to be safe. This PR should also have `0.4.10rc3` set as the version, which I missed and will add now

Reply:
https://github.com/basetenlabs/truss/actions/runs/5425596107 ✅